pkgs/tools/filesystems/bindfs/default.nix

```diff
@@ -20,6 +20,6 @@ stdenv.mkDerivation rec {
     homepage    = "https://bindfs.org";
     license     = lib.licenses.gpl2Only;
     maintainers = with lib.maintainers; [ lovek323 lovesegfault ];
-    platforms   = lib.platforms.unix;
+    platforms   = lib.platforms.unix ++ lib.platforms.darwin;
```

Comment:
Isn't unix linux and darwin plus some others combined?

Reply:
Fixed, was confused by error that it's not supported on darwin when tried to install, but it was probably about fuse dependency.